src/wp-admin/includes/class-wp-filesystem-direct.php

```diff
@@ -487,7 +487,13 @@ public function mtime( $file ) {
 	 * @return int|false Size of the file in bytes on success, false on failure.
 	 */
 	public function size( $file ) {
-		return @filesize( $file );
+		$filesize = wp_filesize( $file );
+
+		if ( 0 === $filesize ) {
+			return false;
+		}
+
+		return $filesize;
```

Comment:
All these lines could be turned into `return wp_filesize( $file ) ?: false;`, which significantly reduces the number of opcodes, making this method much faster.

If only WPCS would permit... 😞